### PR TITLE
Zach

### PR DESCRIPTION
**Pull request by Pixouls**
Zach Zach

### CODE REVIEW

File: 1.3 Data Types/readme.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+
++ Doubles
+
+   Doubles are a form of floating point data that contains 8 bytes. You can use any numberic value and it can be fractional with decimals. It has twice the percision of a float. The default value will be 0d.
+
++ Character `char`
+
+   Characters are a form of character data that contains 2 bytes. You can use any unicode character and it will be case sensitive. Therefore you have all 26 letters of the alphabet, lower and uppercase.    
+
++ Booleen
```
Comment (antoinegag) [inline 1.3 Data Types/readme.md]: Spelled Boolean

File: 1.3 Data Types/readme.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+
+   Booleens are a form of true/false data that contains 1 byte. You can either set this value to true or false, with false being the default.
```
Comment (antoinegag) [inline 1.3 Data Types/readme.md]: Boolean again

File: 1.3 Data Types/readme.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+
+   Floats are a form of floating point data that contains 4 bytes. You can use any numeric value and it can be fractional with decimals. The default value will be 0f.
+
++ Doubles
+
+   Doubles are a form of floating point data that contains 8 bytes. You can use any numberic value and it can be fractional with decimals. It has twice the percision of a float. The default value will be 0d.
+
++ Character `char`
+
+   Characters are a form of character data that contains 2 bytes. You can use any unicode character and it will be case sensitive. Therefore you have all 26 letters of the alphabet, lower and uppercase.
```
Comment (antoinegag) [inline 1.3 Data Types/readme.md]: I think saying 16 bits would be better (opposed to 2 Bytes)

Comment (antoinegag) [inline 1.3 Data Types/readme.md]: Chars can be more than just letters, you have symbols as well.

**Review by firecrackerlinux — APPROVED** [review]
Fixed some small details, converted all bytes to bites, added a few code examples. Will add a lot more code examples in the second stage of revision but for now it's good.

**Review by antoinegag — APPROVED** [review]

**Pull request merged**